scipy/optimize/_optimize.py

```diff
@@ -3358,8 +3361,21 @@ def brute(func, ranges, args=(), Ns=20, full_output=0, finish=fmin,
         `multiprocessing.Pool.map` for evaluating the grid in parallel.
         This evaluation is carried out as ``workers(func, iterable)``.
         Requires that `func` be pickleable.
-
-        .. versionadded:: 1.3.0
+    integrality : 1-D array, optional
+        For each decision variable, a boolean value indicating whether the
```

Comment:
I've used independent variable elsewhere in `optimize`. It'd be good to standardise on either decision or independent, at least within the optimize module.